packages/sui-ssr/README.md

```diff
@@ -309,6 +310,117 @@ SOME_OTHER_ENV_VAR: https://pre.somedomain.com/contact
 SUI-SSR allows 301 redirects in server side rendering when combined with SUI-REACT-INITIAL-PROPS.
 Check out its [documentation](https://github.com/SUI-Components/sui/tree/master/packages/sui-react-initial-props#response-2) to get detailed information and an implementation example.
 
+## Third Parties
+
+It is very likely that for performance reasons you will want to put the third party scripts directly into the index.html of your page.
+
+Although there is nothing wrong with that, you might be interested in measuring the performance of your site, without loading all these scripts. To do this, you would have to mark them with an HTML comment so that they can be removed from the server response, if the request is made with a QueryParam that matches the value set in `queryDisableThirdParties` in your application's sui-ssr configuration.
+
+If this were your `src/index.html` file:
+
+```html
+<html>
+<head>
+  <link rel="preconnect dns-prefetch" href="<%= CDN %>">
+  <!--THIRD_PARTY--><link rel="preconnect dns-prefetch" href="//c.dcdn.es">
+  <!--THIRD_PARTY--><link rel="dns-prefetch" href="//www.google.es">
+  <!--THIRD_PARTY--><link rel="dns-prefetch" href="//www.google.com">
+  <!--THIRD_PARTY--><link rel="dns-prefetch" href="//www.googletagmanager.com">
+
+  <!-- ShellAPP -->
+  <% if (css && vendor && app) { %>
+    <link as="style" rel="preload" href="<%= css %>">
+    <link as="script" rel="preload" href="<%= vendor.entry %>">
+    <link as="script" rel="preload" href="<%= app.entry %>">
+  <% } %>
+
+  <!-- ThridPartyScripts -->
+
+  <!-- Advertisement -->
+  <!--THIRD_PARTY--><link as="script" importance="low" rel="preload" href="<%= utagScript %>">
+  <!--THIRD_PARTY--><link as="script" importance="low" rel="preload" href="<%= openAdsScript %>">
+
+  <!-- Load 3th parties and ShellAPP -->
+  <% if (vendor && app) { %>
+    <script defer importance="high" src="<%= vendor.entry %>"></script>
+    <script defer importance="high" src="<%= app.entry %>"></script>
+  <% } %>
+
+  <!--THIRD_PARTY--><script defer importance="high" src="<%= utagScript %>"></script>
+  <!--THIRD_PARTY--><script defer importance="low" src="<%= openAdsScript %>"></script>
+  
+</head>
+
+<body>
+  <div id="app" class="app">
+    <!-- APP -->
+  </div>
+</body>
+
+</html>
+```
+
+and this is a fragment of his sui-ssr configuration in your package.json
+
+```json
+{
+  "config": {
+    "sui-ssr": {
+      "queryDisableThirdParties": "disable-third-parties",
+    }
+  }
+}
+```
+
+by making a request like this: GET /?disable-third-parties
+
+The sui-ssr response would be an HTML like the following:
+
+
+```html
+<html>
+<head>
+  <link rel="preconnect dns-prefetch" href="<%= CDN %>">
+  <!--THIRD_PARTY-->
```

Comment:
Why we would want to keep this comment?

Reply:
Because delete the tag, not the comment.

I could have done it, but I left it because I didn't see how much it would add to removing it or leaving it. If it's true that it helped me in development to quickly identify what "mode" I was in.

Maybe it's something that also helps in production when you have to see at a glance if you're deleting third party scripts correctly.